src/flight_paths.py

```diff
@@ -0,0 +1,64 @@
+import requests
```

Comment:
This is not a part of the standard library. If you're going to include packages that aren't a part of the standard library, you really should include them in your `setup.py` as a part of `install_requires`.